clients/drcachesim/simulator/snoop_filter.h

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
Please add copyright and license header comments.

Reply:
Added.